src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +13,32 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+
+    public static final String LETTER = "w";
+
+    String[] readFromFile(String fileName) {
+
+        StringBuilder result = new StringBuilder();
+        
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            String nextLine;
+            while ((nextLine = br.readLine()) != null) {
+                String[] words = nextLine.split(" ");
+                for (String str : words) {
+                    if (str.toLowerCase().startsWith(LETTER)) {
+                        result.append(str.toLowerCase()
+                                .replaceAll("\\W", "") + " ");
+                    }
+                }
+            }
```

Comment:
```suggestion
                String[] words = nextLine.toLowerCase()
                        .replaceAll("\\W+", " ")
                        .split(" ");
                for (String str : words) {
                    if (str.startsWith(LETTER)) {
                        result.append(str).append(" ");                      );
                    }
                }
            }
```

Reply:
we will make this operation ones per whole line instead of making it for each word separately